src/components/styles/book.module.css

```diff
@@ -0,0 +1,86 @@
+.book_container {
+  background-color: #fff;
+  margin: 10px auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 1rem 0.5rem;
+  border: solid 1px #e8e8e8;
+  border-radius: 4px;
+  color: #121212;
+}
+
+.book_info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  width: 20%;
+}
+
+.book_info h3 {
+  font-size: 1.5rem;
+}
+
+.book_info h4 {
+  font-size: 13px;
+  color: #0290ff;
+}
```

Comment:
You've done great styling the Bookstore app so far, but there are a few changes you should work on to match your design with the original design given. 
I will be listing the changes below and on the screenshots, I marked the changes you should improve on 👍 

- [ ] Kindly add more space from the navigation bar to the list of books in your app as it is currently cluttered. Give some breathing space from the navigation bar to the first book in the list of boos.
- [ ] In the left part of each book, kindly show the author's name on top of the book's title. For the original design, it is the category of the book but since in your case you are using the author, please show the author on top of the book title and make it a bolded light gray text to not compete with the title text that's why you should make it light gray. 👍 
- [ ] The input fields should have a light gray border on the form for adding a book. 
- [ ] The Abb Book button should have no border or if you want to stick to having a border, you should make the border color match the button's blue background color. 
- [ ] Currently, the current chapter you are displaying on the right part of each book is all in lowercase, to match the design, kindly make the text uppercase and a light graded text. 

**The Original Design with highlights on changes to be fixed**

![image](https://user-images.githubusercontent.com/50941074/179320790-01fa0741-f9d0-4338-86b5-80b44c027d36.png)

**Your Design with changes highlighted that need fixing**

![image](https://user-images.githubusercontent.com/50941074/179321163-b6120e74-bdcf-47c6-8b0d-5f9b9a67c18b.png)